platform/viewer/src/components/ViewportGrid/ViewportGrid.js

```diff
@@ -32,6 +32,11 @@ const ViewportGrid = function(props) {
 
   const ViewportPanes = layout.viewports.map((layout, viewportIndex) => {
     const displaySet = viewportData[viewportIndex];
+
+    if (!displaySet) {
+      return null;
+    }
```

Comment:
I wonder if this is possible because we don't clear `viewportSpecificData` when our `numRows` or `numColumns` are updated?